zipline/data/adjustments.py

```diff
@@ -209,9 +211,9 @@ def load_pricing_adjustments(self, columns, dates, assets):
     def get_adjustments_for_sid(self, table_name, sid):
         t = (sid,)
         c = self.conn.cursor()
-        adjustments_for_sid = c.execute(
-            "SELECT effective_date, ratio FROM %s WHERE sid = ?" %
-            table_name, t).fetchall()
+        c.execute(f'SELECT effective_date, ratio FROM {table_name} WHERE sid = {sid}')
```

Comment:
switching to format-string instead of prepared-queries of course makes us vulnerable for injections. however, the string was partly formated anyway, ruling out this advantage. on the other hand, using string-format alone makes it possible to use sqlite and psycopg2 in parallel